contributors/devel/api-conventions.md

```diff
@@ -317,14 +319,14 @@ following fields, but must contain at least `type` and `status` fields:
   // +optional
   LastHeartbeatTime  unversioned.Time  `json:"lastHeartbeatTime,omitempty" description:"last time we got an update on a given condition"`
```

Comment:
Agree -- I can't think of an API that we actually have that field in, and was surprised to see it.